packages/ketcher-core/src/application/render/restruct/rebond.ts

```diff
@@ -57,18 +57,178 @@ class ReBond extends ReObject {
     return true
   }
 
+  static bondRecalc(bond: ReBond, restruct: ReStruct, options: any): void {
+    const render = restruct.render
+    const atom1 = restruct.atoms.get(bond.b.begin)
+    const atom2 = restruct.atoms.get(bond.b.end)
+
+    if (
+      !atom1 ||
+      !atom2 ||
+      bond.b.hb1 === undefined ||
+      bond.b.hb2 === undefined
+    ) {
+      return
+    }
+
+    const p1 = Scale.obj2scaled(atom1.a.pp, render.options)
+    const p2 = Scale.obj2scaled(atom2.a.pp, render.options)
+    const hb1 = restruct.molecule.halfBonds.get(bond.b.hb1)
+    const hb2 = restruct.molecule.halfBonds.get(bond.b.hb2)
+
+    if (!hb1?.dir || !hb2?.dir) return
+
+    hb1.p = shiftBondEnd(atom1, p1, hb1.dir, 2 * options.lineWidth)
+    hb2.p = shiftBondEnd(atom2, p2, hb2.dir, 2 * options.lineWidth)
+    bond.b.center = Vec2.lc2(atom1.a.pp, 0.5, atom2.a.pp, 0.5)
```

Comment:
May be you can add a comment for `lc2` (where it is defined)? 
For me it is still a mystery, what it does.

Reply:
@Nitvex I actually just moved the bondRecalc function so it is a static method now, its a bit of a mystery for myself what lc2 does 😄 I'll try to figure it out and a dd a comment

Reply:
@KonstantinEpam23 @Nitvex I thought lc was the abbreviation of [Linear Combination](https://en.wikipedia.org/wiki/Linear_combination). The difference between function `lc` and `lc2` was the number of arguments they could receive. `lc` could combine lots of vectors, and `lc2` could only combine 2 vectors, so functionality-wise `lc2` could be replaced by `lc`.